raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -129,16 +130,25 @@ public String toString() {
                 QUORUM_LINGER_MS_DOC);
     }
 
-    public RaftConfig(Properties props) {
-        super(CONFIG, props);
-    }
+    private final int requestTimeoutMs;
+    private final int retryBackoffMs;
+    private final int electionTimeoutMs;
+    private final int electionBackoffMaxMs;
+    private final int fetchTimeoutMs;
+    private final int appendLingerMs;
 
-    public RaftConfig(Map<String, Object> props) {
-        super(CONFIG, props);
+    public RaftConfig(Map<?, ?> props) {
+        this(props, true);
     }
 
     protected RaftConfig(Map<?, ?> props, boolean doLog) {
         super(CONFIG, props, doLog);
+        requestTimeoutMs = getInt(QUORUM_REQUEST_TIMEOUT_MS_CONFIG);
+        retryBackoffMs = getInt(QUORUM_RETRY_BACKOFF_MS_CONFIG);
+        electionTimeoutMs = getInt(QUORUM_ELECTION_TIMEOUT_MS_CONFIG);
+        electionBackoffMaxMs = getInt(QUORUM_ELECTION_BACKOFF_MAX_MS_CONFIG);
+        fetchTimeoutMs = getInt(QUORUM_FETCH_TIMEOUT_MS_CONFIG);
+        appendLingerMs = getInt(QUORUM_LINGER_MS_CONFIG);
```

Comment:
Is it worth storing a field for voter connections as well to be consistent?

Reply:
Yea, I think that's reasonable. Added.